### PR TITLE
feat(aio): filetree and top bar hover styles

### PR DESCRIPTION
**Pull request by sjtrimble**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: aio/src/styles/_constants.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@ $code-font: "Droid Sans Mono", monospace;
 
 // COLOR PALETTE
 $blue: #1976D2;
+$accentblue: #1E88E5;
```
Comment (gkalpak) [inline aio/src/styles/_constants.scss]: Is this necessary?

**Review by gkalpak — CHANGES_REQUESTED** [review]
It is a little confusing that you can't click on the highlighted thing (the `<li>`) to navigate, but you have to click on the text (the `<a>`). 

It would be better if the highlighted "thing" (the `<li>`) got a pointer cursors and was clickable.

File: aio/src/styles/_constants.scss (hunk 1 above)
Comment (sjtrimble) [inline aio/src/styles/_constants.scss]: Yes - this is the hover color in the _top-menu.scss file for the li's. I've been meaning to add it for additional color styling (considering changing all links to change to this on hover).

**Comment by petebacondarwin** [comment]
The padding seems to be slightly out on the top bar items...

screen shot 2017-04-14 at 21 01 44

**Comment by sjtrimble** [comment]
@petebacondarwin @gkalpak I just cleaned up the padding on the top bar, and not the entire highlighted box should be clickable.

**Review by petebacondarwin — APPROVED** [review]
Much better!

**Pull request merged**